plugins/pulp_rpm/plugins/importers/yum/associate.py

```diff
@@ -57,7 +57,8 @@ def associate(source_repo, dest_repo, import_conduit, config, units=None, solver
         solver = pulp_solv.Solver(
             source_repo,
             target_repo=dest_repo,
-            conservative=config.get(constants.CONFIG_RECURSIVE_CONSERVATIVE)
+            conservative=config.get(constants.CONFIG_RECURSIVE_CONSERVATIVE),
+            ignore_missing=False
```

Comment:
This may need to be reverted later, not sure yet. In practice, I've gotten the correct results back from the solver even though it's reporting problems such as 

```
'nothing provides rtld(GNU_HASH) needed by silver-0:1.0.5-x86_64.x86_64',
 'nothing provides libpthread.so.0()(64bit) needed by silver-0:1.0.5-x86_64.x86_64',
 'nothing provides libpthread.so.0(GLIBC_2.2.5)(64bit) needed by silver-0:1.0.5-x86_64.x86_64',
 'nothing provides libcrypto.so.1.1()(64bit) needed by silver-0:1.0.5-x86_64.x86_64',
 'nothing provides libdl.so.2()(64bit) needed by silver-0:1.0.5-x86_64.x86_64',
 'nothing provides libdl.so.2(GLIBC_2.2.5)(64bit) needed by silver-0:1.0.5-x86_64.x86_64',
```

Our dummy-solvable-injecting code is what is causing the crashes, and disabling it fixes them and so far seems to give perfectly correct results, so unless this is proven false later I think we should proceed.

I can move it into a different commit, though, if you want.

Reply:
@dralley , +1 to mark it somehow. Maybe an inline comment is enough? So someone else and not only you :) would be able to find the spot.